doc/docs.md

```diff
@@ -2954,6 +2956,37 @@ Full list of builtin options:
 | `gnu`, `hpux`, `haiku`, `qnx` | `cplusplus`       | `big_endian`          | |
 | `solaris`, `linux_or_macos`   | | | |
 
+### Environment specific files
+
+If a file has environment-specific suffix, it will only be compiled that environment
```

Comment:
"If a file has an environment-specific suffix, it will only be compiled for that environment."